cp-connector/pkg/controlplane/cp_eventsender.go

```diff
@@ -0,0 +1,28 @@
+package controlplane
+
+import (
+	"context"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/keptn/go-utils/pkg/lib/v0_2_0"
+)
+
+type CPEventSender struct {
+	Sender EventSender
+}
+
+func (e *CPEventSender) SendEvent(event cloudevents.Event) error {
+	keptnEvent, err := v0_2_0.ToKeptnEvent(event)
+	if err != nil {
+		return err
+	}
+	return e.Sender(keptnEvent)
```

Comment:
I would avoid code duplication here
```suggestion
	return Send(context.TODO(), keptnEvent)
```